test/test.sh

```diff
@@ -1,13 +1,13 @@
-#! /bin/bash
+#!/usr/bin/env bash
 
 # wgetpaste test script
 # Exit code: number of mismatched downloads or 1 for general failure
-# Copyright (C) 2021  xxc3nsoredxx
+# Copyright (C) 2022 Oskari Pirhonen <xxc3ncoredxx@gmail.com>
```

Comment:
I'm ok with this, but shouldn't it be "2021-2022" since that's when it was first created?